caas/kubernetes/provider/bootstrap.go

```diff
@@ -327,9 +330,9 @@ func (c *controllerStack) Deploy() (err error) {
 			return false
 		}
 	}
-	if isDone() {
-		return bootstrap.Cancelled()
-	}
+	// if isDone() {
```

Comment:
Should this just be removed?

Reply:
Uncommented.
This broke the test because of context.Context was not mocked.